app/sign.js

```diff
@@ -174,29 +177,35 @@ const handleSignUtxo = function(recoveryRequest, key, skipConfirm) {
 
     // Derive signing key from chain path
     const derivedHDNode = backupKeyNode.derivePath(input.chainPath);
-    console.log(`Signing input ${ i + 1 } of ${ recoveryRequest.inputs.length } with ${ derivedHDNode.neutered().toBase58() } (${ input.chainPath })`);
+    console.log(`Signing input ${ i + 1 } of ${ recoveryRequest.inputs.length } 
```

Comment:
I think this will keep the new line in there which may not be what you want. Not a blocker though.

Reply:
good catch, fixed